mobile-app/components/PlayableCell.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Polygon } from 'react-native-svg';
+import type { Cell } from '../../web-app/src/common/gameState';
+import { DEFAULT_CELL_COLOR, PLAYER_1_CELL_COLOR, PLAYER_2_CELL_COLOR } from "../utils";
+
+interface PlayableCellData {
+  svgPoints: string;
+  strokeColor: string;
+  cellValue: Cell;
+  onCellPress: () => void;
+}
+
+export default function PlayableCell(props: PlayableCellData) {
+  return (
+    <Polygon
+      points={props.svgPoints}
+      stroke={props.strokeColor}
+      fill={getPawnColor(props.cellValue)}
+      onClick={() => props.onCellPress()}
+      onPress={() => props.onCellPress()}
+    />)
+}
+
+function getPawnColor(cellValue: Cell) {
+  return cellValue.value == "white" ? PLAYER_1_CELL_COLOR : cellValue.value == "black" ? PLAYER_2_CELL_COLOR : DEFAULT_CELL_COLOR;

```

Comment:
the "white" and "black" strings should be constants. Also, use triple equal.

Reply:
Same answer than for Clement(https://github.com/marmelab/Hex/pull/37#discussion_r811962308) we are using an union type here so we have some protection against typing invalid values. Is it enough ?